Packs/SIGNL4/Integrations/SIGNL4/SIGNL4.yml

```
@@ -0,0 +1,53 @@
+category: Messaging and Conferencing
+commonfields:
+  id: SIGNL4
+  version: -1
+configuration:
+- display: Team or Integration Secret
+  name: secret
+  hiddenusername: true
+  type: 4
```

```suggestion
  type: 9
```
https://xsoar.pan.dev/docs/integrations/parameter-types#authentication